docs/source/specs/openapi.json

```diff
@@ -1444,275 +1440,7 @@
         }
       }
     },
-    "/policies/": {
```

Comment:
I wonder if we need to be concerned with any clients already build against this doc _with_ policies? Probably worth at least an email on official deprecation.

Reply:
Good thought, will handle that Monday.